charts/cloudzero-agent/values.yaml

```diff
@@ -67,7 +67,7 @@ validator:
   name: env-validator
   image:
     repository: ghcr.io/cloudzero/cloudzero-agent-validator/cloudzero-agent-validator
-    tag: latest
+    tag: 0.4.0
```

Comment:
This is the latest version of the validator before implementing Dynamic service discovery.